src/platform/Tizen/BLEManagerImpl.cpp

```diff
@@ -926,10 +1007,36 @@ CHIP_ERROR BLEManagerImpl::_Init()
     return err;
 }
 
-void BLEManagerImpl::_Shutdown()
+CHIP_ERROR BLEManagerImpl::_Shutdown()
```

Comment:
Why this change? There is `void BLEManager::Shutdown()`, so the `_Shutdown()` in the Impl class should also be void.